keras_nlp/models/bert/bert_masked_lm.py

```diff
@@ -139,6 +140,12 @@ def __init__(
         # All references to `self` below this line
         self.backbone = backbone
         self.preprocessor = preprocessor
+        self.compile(
+            loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True),
+            optimizer=keras.optimizers.Adam(5e-5),
+            metrics=keras.metrics.SparseCategoricalAccuracy(),
```

Comment:
I believe this should be `weighted_metrics` as we pass sample weights in the preprocessing.

Reply:
that's great 😀